### DISCUSSION TITLE
Not showing as a collaborator

### DISCUSSION THREAD

**Discussion by saharmaisha** [community/community#81502] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was a collaborator on a project and have pushed changes, but I am not showing up as a collaborator. What should I do? My graph even says that I do not have any contributions for that month which is false. I checked by running `git config user.email` and saw that my personal email that I use for Github is indeed linked to the project that I worked on. How do I fix this?

**Comment by BilalBoudjema** [comment]
To resolve the issue of your contributions not showing up on GitHub, try these steps:

1. Check Email in Commits: Use git log to verify that the email used in your commits matches the one associated with your GitHub account.

2. Add Email to GitHub: If the commit email isn't linked to your GitHub account, add it in your GitHub account settings.

3. Merge Branches: Ensure your commits are in the branch that GitHub tracks for contributions, typically the main or master branch.

4. Repository Settings: Check if the repository is private or archived, as it affects contribution visibility.

5. Forked Repository: Contributions to forks don't count unless merged into the parent repository.

6. Link Past Commits: If necessary, use GitHub's Email Rewrite Tool to link your commit email to your GitHub account.

7. Push Changes: Make sure you've pushed your changes to the correct branch.

8. Contact Support: If issues persist, contact GitHub Support for help.

9. Make sure your local Git email configuration aligns with your GitHub account email.
